Dockerfile

```diff
@@ -1,12 +1,15 @@
-# syntax = docker/dockerfile:experimental
-#
-# NOTE: To build this you will need a docker version > 18.06 with
-#       experimental enabled and DOCKER_BUILDKIT=1
-#
-#       If you do not use buildkit you are not going to have a good time
+# syntax=docker/dockerfile:1
+
+# NOTE: Building this image requires a minimum docker version of either:
+#  - > 18.06 with `experimental` enabled and ENV `DOCKER_BUILDKIT=1`
+#  - >= 23.0 releases (Feb 2023) use BuildKit by default
 #
-#       For reference:
-#           https://docs.docker.com/develop/develop-images/build_enhancements/
+# For reference:
+# - https://docs.docker.com/build/buildkit/#getting-started
+# - https://docs.docker.com/build/dockerfile/frontend/#stable-channel
+# - https://docs.docker.com/engine/release-notes/23.0/
+# - https://hub.docker.com/layers/docker/dockerfile/1.0/images/sha256-92f5351b2fca8f7e2f452aa9aec1c34213cdd2702ca92414eee6466fab21814a?context=explore
```

Comment:
Can this one be replaces with a simple `Building this image requires minimum docker version of 23.0?

Reply:
v23 as a minimum is what `docker-mailserver` mandates (_a project I maintain_), since rarely do we have users that need to build the Docker image themselves, most just use the published image directly.

We've not had any notable issues regarding that, but our own docs page for building the image do link to the Docker docs regarding `DOCKER_BUILDKIT=1` for those that may need it. Our `Dockerfile` itself has no comments like the above however.